python/lsst/validate/drp/matchreduce.py

```diff
@@ -282,6 +282,10 @@ def _loadAndMatchCatalogs(repo, dataIds, matchRadius,
     srcVis = SourceCatalog(newSchema)
 
     for vId in dataIds:
+        if not butler.datasetExists('src', vId):
+            print("Could not find source catalog for ", vId)
```

Comment:
I suggest: print(f"Could not find source catalog for {vId}; skipping this dataId.")
... especially if this line could get printed out many times; I would personally find it annoying to have lots of mostly redundant "Skipping this dataId" messages cluttering the logs. If it's only a few then it's not a big deal.